websvc.metro.lib/nbproject/project.properties

```diff
@@ -22,14 +22,18 @@ jnlp.indirect.jars=\
     modules/ext/metro/webservices-extra-api.jar,\
     modules/ext/metro/webservices-rt.jar
 
-release.external/metro-2.0.zip!/webservices-api.jar=modules/ext/metro/webservices-api.jar
-release.external/metro-2.0.zip!/webservices-extra.jar=modules/ext/metro/webservices-extra.jar
-release.external/metro-2.0.zip!/webservices-tools.jar=modules/ext/metro/webservices-tools.jar
-release.external/metro-2.0.zip!/webservices-extra-api.jar=modules/ext/metro/webservices-extra-api.jar
-release.external/metro-2.0.zip!/webservices-rt.jar=modules/ext/metro/webservices-rt.jar
+release.external/webservices-api-2.3.1.jar=modules/ext/metro/webservices-api.jar
+release.external/webservices-extra-2.3.1.jar=modules/ext/metro/webservices-extra.jar
+release.external/webservices-tools-2.3.1.jar=modules/ext/metro/webservices-tools.jar
+release.external/webservices-extra-api-2.3.1.jar=modules/ext/metro/webservices-extra-api.jar
+release.external/webservices-rt-2.3.1.jar=modules/ext/metro/webservices-rt.jar
 
 # Metro Javadoc
-release.external/metro-1.4-doc.zip=docs/metro-1.4-doc.zip
+release.external/webservices-api-2.3.1-javadoc.jar=docs/webservices-api-2.3.1-javadoc.jar
+release.external/webservices-extra-2.3.1-javadoc.jar=docs/webservices-extra-2.3.1-javadoc.jar
+release.external/webservices-tools-2.3.1-javadoc.jar=docs/webservices-tools-2.3.1-javadoc.jar
+release.external/webservices-extra-api-2.3.1-javadoc.jar=docs/webservices-extra-api-2.3.1-javadoc.jar
+release.external/webservices-rt-2.3.1-javadoc.jar=docs/webservices-rt-2.3.1-javadoc.jar
```

Comment:
Please also update: `websvc.metro.lib/src/org/netbeans/modules/websvc/metro/lib/metro.xml`. That file declares the library in netbeans (Tools -> Library). The javadoc files need to be added to the javadoc volumne (see line 39 for the reference to the metro doc). As the binaries are not versioned, I would also drop the version from the java docs.

When you update `metro.xml` I would also udate the maven dependencies to:

```
org.glassfish.metro:webservices-api:2.3.1
org.glassfish.metro:webservices-extra:2.3.1
org.glassfish.metro:webservices-extra-api:2.3.1
org.glassfish.metro:webservices-rt:2.3.1
org.glassfish.metro:webservices-tools:2.3.1
```

Reply:
Corrected, Thanks